### PR TITLE
Harshad_Assignment_22_Done_30mins

### PR DESCRIPTION
**Pull request by HarshadBJun20**
-> Assignment 22 Done.
-> Program 1 screenshot of output:
image

-> Program 2 screenshot of output:
image

### CODE REVIEW

File: src/harshadBJun20/Assignment22/CheckStringPalindromeOrNot.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Output : Given string radar is palindrome.
+
+Input: technocredits
+Output: Given string technocredits is not palindrome.*/
+
+public class CheckStringPalindromeOrNot {
+	
+	void checkPalindrome(String str) {
+		String str1="";
+		for(int i=str.length()-1;i>=0;i--) {
```
Comment (AasthaJJun20) [inline src/harshadBJun20/Assignment22/CheckStringPalindromeOrNot.java:15]: Please use variable name as index, rest the code is perfect!

**Review by AasthaJJun20 — APPROVED** [review]
Code looks good to me.

File: src/harshadBJun20/Assignment22/CheckStringPalindromeOrNot.java (hunk 1 above)
Comment (HarshadBJun20) [reply src/harshadBJun20/Assignment22/CheckStringPalindromeOrNot.java:15]: Ok.. I will use index as variable in future prog.

File: src/harshadBJun20/Assignment22/CheckNumberPalindromeOrNot.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Input: number = 45654
+Output : Given number 45654 is palindrome.
+
+Input: number = 56789
+Output : Given number 56789 is not palindrome.
+*/
+public class CheckNumberPalindromeOrNot {
+	
+	void checkPalindrome(int num) {
+		int seperateNum=0, reversedNum = 0;
```
Comment (KrishnaKTechnocredits) [inline src/harshadBJun20/Assignment22/CheckNumberPalindromeOrNot.java:15]: No need to use extra variable seperateNum.

File: src/harshadBJun20/Assignment22/CheckNumberPalindromeOrNot.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+		int number =num;
+		while(num>0) {
+			seperateNum = num % 10;
+			reversedNum = reversedNum * 10 + seperateNum;
+			num = num / 10;
```
Comment (KrishnaKTechnocredits) [inline src/harshadBJun20/Assignment22/CheckNumberPalindromeOrNot.java:20]: reversedNum = (reversedNum * 10) + (num % 10);
num = num / 10;

File: src/harshadBJun20/Assignment22/CheckNumberPalindromeOrNot.java (hunk 2 above)
Comment (HarshadBJun20) [reply src/harshadBJun20/Assignment22/CheckNumberPalindromeOrNot.java:15]: ok

File: src/harshadBJun20/Assignment22/CheckNumberPalindromeOrNot.java (hunk 3 above)
Comment (HarshadBJun20) [reply src/harshadBJun20/Assignment22/CheckNumberPalindromeOrNot.java:20]: yes.. got it

**Pull request merged**